pxr/usd/usd/testenv/testCoreUsdValidators.cpp

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+
+#include "pxr/usd/usd/validator.h"
+#include "pxr/usd/usd/validationError.h"
+#include "pxr/usd/usd/validatorTokens.h"
+#include "pxr/usd/usd/validationRegistry.h"
+
+#include <iostream>
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+static
+void TestCoreUsdStageMetadata()
+{
+
+    // Get stageMetadataChecker
```

Comment:
This is good, but I will also add a test of a valid stage which provides a defaultPrim.

Reply:
Updated